Data-Analytics/Spark/k8s/DataProcessTransferFts-JarLocal-3.3.1.yaml

```diff
@@ -8,14 +8,14 @@ spec:
     spark.hadoop.fs.s3a.path.style.access: "true"
     spark.hadoop.fs.s3a.impl: "org.apache.hadoop.fs.s3a.S3AFileSystem"
     spark.hadoop.fs.s3a.connection.ssl.enabled: "true"
-
-    # Needed if s3 server is behind proxy
-    # spark.hadoop.fs.s3a.proxy.host: "web-proxy.corp.hpecorp.net"
-    # spark.hadoop.fs.s3a.proxy.port: "8080"
-
-    # Needed if s3 server certs are not imported to java truststore used by spark
-    "spark.driver.extraJavaOptions": "-Dcom.amazonaws.sdk.disableCertChecking=true"
-    "spark.executor.extraJavaOptions": "-Dcom.amazonaws.sdk.disableCertChecking=true"
+    spark.hadoop.fs.s3a.access.key: <access-key>
+    spark.hadoop.fs.s3a.secret.key: <secret-key>
```

Comment:
For S3 proxy case this can be hardcoded to 's3'

Reply:
Yes, access key should have JWT token, and secret key can have any value iianm, so leave 's3'